es6.js

```diff
@@ -4,7 +4,6 @@ module.exports = {
         "constructor-super": "error",
         "generator-star-spacing": "error",
         "no-class-assign": "error",
-        "no-confusing-arrow": "error",
```

Comment:
Why this removal?

Reply:
Because it's a really dumb rule that blocks on completely legit JS.

Reply:
From ESLint:

```
// The intent is not clear
var x = a => 1 ? 2 : 3;
```

It's a poorly written function but we should be catching those in code review anyway. It blocks  things like this:

```
const fn = a => a ? 'yeet' : 'peet';
```

Which it definitely should not.